terraform/environments/corporate-staff-rostering/ec2_common.tf

```
@@ -40,16 +40,45 @@ resource "aws_ssm_document" "cloud_watch_agent" {
   )
 }
 
-resource "aws_ssm_document" "ami_build" {
+resource "aws_ssm_document" "ami_build-command" {
   name            = "ami-build"
   document_type   = "Command"
   document_format = "YAML"
-  content         = file("./ssm-documents/ami-build.yaml")
+  content         = file("./ssm-documents/ami-build-command.yaml")
 
   tags = merge(
     local.tags,
     {
-      Name = "ami-build"
+      Name = "ami-build-command"
     },
   )
-}
\ No newline at end of file
+}
+
+resource "aws_ssm_document" "ami_build-automation" {
```

Underline instead of dash.
```suggestion
resource "aws_ssm_document" "ami_build_automation" {
```